test/scripts/e2e_subs/app-version.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+filename=$(basename "$0")
+scriptname="${filename%.*}"
+date "+${scriptname} start %Y%m%d_%H%M%S"
+
+set -e
+set -x
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+
+TEAL=test/scripts/e2e_subs/tealprogs
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+
+APPID=$(${gcmd} app create --creator "$ACCOUNT" --approval-prog=${TEAL}/approve-all.teal --clear-prog=${TEAL}/approve-all.teal | grep Created | awk '{ print $6 }')
+
+ACCOUNTB=$(${gcmd} account new|awk '{ print $6 }')
+${gcmd} clerk send -f "$ACCOUNT" -t "$ACCOUNTB" -a 1000000
+
+# Now call from a different account, reject-version=0 allows all
+${gcmd} app call --app-id="$APPID" --from="$ACCOUNTB" --reject-version 0
+
+# reject-version=1 allows because version is currently 1
```

Comment:
This should reject right (reject version >= reject-version)

Reply:
Typo in the comment, fixing. The version is 0, because it was just created.  The test a few lines down correctly checks that it fails when the app version is 1 and `--reject-version=1` is used.